master/manifests/site.pp

```diff
@@ -244,59 +169,58 @@
     notify => Service['jenkins'],
 }
 
-jenkins::plugin {
-  'purge-build-queue-plugin': ;
-}
-
+jenkins::plugin { 'purge-build-queue-plugin': ; }
 # required by mercurial
-jenkins::plugin {
-  'scm-api': ;
-}
-
+jenkins::plugin { 'scm-api': ; }
 # required by groovy-postbuild
-jenkins::plugin {
-  'script-security': ;
-}
-
-jenkins::plugin {
-  'ssh-agent': ;
-}
-
-jenkins::plugin {
-  'subversion': ;
-}
-
-jenkins::plugin {
-  'systemloadaverage-monitor': ;
-}
-
-jenkins::plugin {
-  'swarm':
+jenkins::plugin { 'script-security': ; }
+# required by groovy-postbuild
+# required by groovy-postbuild
+jenkins::plugin { 'durable-task': ; }
+# required by groovy-postbuild
+jenkins::plugin { 'ssh-agent': ; }
+# needed for ssh-agent
+jenkins::plugin { 'ssh-credentials': ; }
+jenkins::plugin { 'subversion': ; }
+jenkins::plugin { 'systemloadaverage-monitor': ; }
+jenkins::plugin { 'swarm':
     require => Package['wget'];
 }
+jenkins::plugin { 'timestamper': ; }
+# required by build-timeout
+jenkins::plugin { 'token-macro': ; }
+# required by groovy-postbuild
+jenkins::plugin { 'workflow-api': ; }
+# needed for ssh-agent
+jenkins::plugin { 'workflow-cps': ; }
+# required by groovy-postbuild
+jenkins::plugin { 'workflow-scm-step': ; }
```

Comment:
Where did you find this? I don't see it anywhere in here: https://github.com/jenkinsci/groovy-postbuild-plugin/blob/master/pom.xml